doc/nrf/ug_bt_fast_pair.rst

```diff
@@ -97,6 +97,15 @@ The Bluetooth privacy is selected by the Fast Pair service, but you must make su
 
 See the official `Fast Pair Advertising`_ documentation for detailed information about the requirements related to discoverable and not discoverable advertising.
 
+Battery Notification extension
+------------------------------
+
+The `Fast Pair`_ documentation specifies `Fast Pair Battery Notification extension`_.
```

Comment:
Maybe we could move this under :c:func:`bt_fast_pair_adv_data_size`, :c:func:`bt_fast_pair_adv_data_fill` section.

We could simply inform that battery notification is Fast Pair extension and if enabled in the advertising configuration, user needs to provide battery values for all of the components using FP battery set API.

Reply:
The battery mode needs to be always specified  in the advertising configuration, so maybe we could introduce the information in place where we inform about advertising data generation.